runtime/doc/editorconfig.txt

```diff
@@ -86,4 +86,11 @@ Example: >lua
       vim.b[bufnr].foo = val
     end
 <
+
+EditorConfig overrides filetype plugins           *ftplugin-vs-editorconfig*
```

Comment:
I'm also not sure about this help tag. Maybe `editorconfig` should come first? Maybe `*-and-*`?

Reply:
editorconfig-vs-ftplugin would be better.